### PR TITLE
Chris1

### PR DESCRIPTION
**Pull request by chris529**
grammar and punctuation updates

### CODE REVIEW

File: modules/ROOT/pages/index.adoc [hunk 1]
```diff
@@ -77,22 +77,22 @@ Before running Lisk SDK, the following dependencies need to be installed in orde
 
 You can find further details on installing these dependencies in our xref:setup.adoc[pre-installation setup guide].
 
-Mind, that you need to create a database before.
-The default database name is `lisk_dev`, so for the development purposes, a command `createdb lisk_dev` will set you up.
-The default database user and password are `lisk` and `password`, they can be both changed in the xref:configuration.adoc[configuration] of Lisk SDK.
+Please be aware that firstly you need to create a database.
+The default database name is `lisk_dev`, so for the development purposes, a command `createdb lisk_dev` must be used for the set up.
```
Comment (Tschakki) [inline modules/ROOT/pages/index.adoc]:
```suggestion
The default database name is `lisk_dev`, so for the development purposes, a command `createdb lisk_dev --owner lisk` must be used for the set up.
```

File: modules/ROOT/pages/index.adoc [hunk 2]
```diff
@@ -158,7 +158,7 @@ app.run() <3>
 });
 ----
 
-<1> Require the lisk-sdk package.
+<1> Require the Lisk-SDK package.
```
Comment (Tschakki) [inline modules/ROOT/pages/index.adoc]: We should either write `lisk-sdk` when referring to the NPM package, or Lisk SDK when we talk about the product in general. "Lisk-SDK" I would avoid if possible.

File: modules/ROOT/pages/index.adoc [hunk 3]
```diff
@@ -136,10 +136,10 @@ npm install lisk-sdk <2>
 ----
 
 <1> Initialize the project by creating a `package.json` file
-<2> Install the lisk-sdk package as dependency of your project.
-Make sure you installed the dependencies of lisk-sdk beforehand.
+<2> Install the Lisk-SDK package as dependency of your project.
```
Comment (Tschakki) [inline modules/ROOT/pages/index.adoc]: We should either write lisk-sdk when referring to the NPM package, or Lisk SDK when we talk about the product in general. "Lisk-SDK" I would avoid if possible.

File: modules/ROOT/pages/index.adoc [hunk 4]
```diff
@@ -136,10 +136,10 @@ npm install lisk-sdk <2>
...
+Ensure the installed dependencies of the Lisk-SDK are completed beforehand.
```
Comment (Tschakki) [inline modules/ROOT/pages/index.adoc]: Same here.

File: modules/ROOT/pages/index.adoc [hunk 5]
```diff
@@ -77,22 +77,22 @@ Before running Lisk SDK, the following dependencies need to be installed in orde
...
-The default database name is `lisk_dev`, so for the development purposes, a command `createdb lisk_dev` will set you up.
-The default database user and password are `lisk` and `password`, they can be both changed in the xref:configuration.adoc[configuration] of Lisk SDK.
+Please be aware that firstly you need to create a database.
+The default database name is `lisk_dev`, so for the development purposes, a command `createdb lisk_dev` must be used for the set up.
+The default database user and password are `lisk` and `password`, they can be both changed in the xref:configuration.adoc[configuration] of the Lisk SDK.
 
 === Installation
 
-The installation of Lisk Alpha SDK is straightforward and limited to getting a single NPM package, `lisk-sdk`, to your Node.js project:
+Installation of the Lisk Alpha SDK is straightforward and limited to getting a single NPM package, `lisk-sdk`, to your Node.js project:
```
Comment (Tschakki) [inline modules/ROOT/pages/index.adoc]: Remove "Alpha" here

File: modules/ROOT/pages/index.adoc [hunk 6]
```diff
@@ -234,17 +234,17 @@ TIP: For information on creating your own custom transaction, see the xref:custo
...
 == How to interact with the network
 
-While your network is up and running, interact with it and the node through the API and use xref:lisk-commander/index.adoc[Lisk Commander] or xref:lisk-elements/index.adoc[Lisk Elements] to create sendable transaction objects.
+While your network is up and running, interact with it and the node through the API, and use xref:lisk-commander/index.adoc[Lisk Commander] or xref:lisk-elements/index.adoc[Lisk Elements] to create sendable transaction objects.
 
 To monitor and explore the network, you may want to set up a monitoring solution like https://github.com/LiskHQ/lisk-explorer[Lisk Explorer].
 
 Depending on the level of customization, Lisk Explorer and Lisk Commander may also require customization to prevent other services from failing.
 
 Another simple way to interact with the network is by connecting it to https://github.com/LiskHQ/lisk-hub[Lisk Hub].
```
Comment (Tschakki) [inline modules/ROOT/pages/index.adoc]: Rename to `Lisk Desktop`
```suggestion
Another simple way to interact with the network is by connecting it to https://github.com/LiskHQ/lisk-desktop[Lisk Desktop].
```

File: modules/ROOT/pages/index.adoc [hunk 7]
```diff
@@ -234,17 +234,17 @@ TIP: For information on creating your own custom transaction, see the xref:custo
...
-Lisk Hub provides a simple and beautiful user interface to create and manage accounts on the network, and also to interact with the network by sending different types of transactions.
+Lisk Hub provides a simple and user-friendly interface to create and manage accounts on the network, and also to interact with the network by sending different types of transactions.
```
Comment (Tschakki) [inline modules/ROOT/pages/index.adoc]:
```suggestion
Lisk Desktop provides a simple and user-friendly interface to create and manage accounts on the network, and also to interact with the network by sending different types of transactions.
```

File: modules/ROOT/pages/lisk-framework/index.adoc [hunk 8]
```diff
@@ -105,35 +105,35 @@ export default class MyModule extends BaseModule {
...
 
 By default, modules will run in the same process as the controller, which loads the module.
-To load a module in a child process, make sure you have `ipc` enabled in the xref:configuration.adoc#_structure[config] and xref:2.1.0@lisk-core::administration/source.adoc#_command_line_options[set the environment variable] `LISK_CHILD_PROCESS_MODULES` with the module alias.
+To load a module in a child process, ensure the `ipc` is enabled in the xref:configuration.adoc#_structure[config] and xref:2.1.0@lisk-core::administration/source.adoc#_command_line_options[set the environment variable] `LISK_CHILD_PROCESS_MODULES` with the module alias.
 
-TIP: If the respective module is using a lot of CPU power, loading a module in a child process can prevent CPU usage bottlenecks.
+TIP: If the respective module is using a high amount of CPU power, loading a module in a child process can prevent CPU usage bottlenecks.
 
-Multiple modules can be defined by using commas like: `LISK_CHILD_PROCESS_MODULES=httpApi,chain`.
+Multiple modules can be defined by using commas, as shown below: `LISK_CHILD_PROCESS_MODULES=httpApi,chain`.
```
Comment (Tschakki) [inline modules/ROOT/pages/lisk-framework/index.adoc]: It's not really below, it's on same line, maybe just remove that
```suggestion
Multiple modules can be defined by using commas: `LISK_CHILD_PROCESS_MODULES=httpApi,chain`.
```

File: modules/ROOT/pages/tutorials/cashback.adoc [hunk 9]
```diff
@@ -105,9 +105,9 @@ Create the file `index.js`, which will hold the logic to initialize and start th
 
 == 2. Configure the application
 
-Next, let’s configure the application, to provide basic information about the application we are going to build.
+The next step  is to provide the basic information about the application to be built.
```
Comment (Tschakki) [inline modules/ROOT/pages/tutorials/cashback.adoc]:
```suggestion
The next step is to provide the basic information about the application to be built.
```

File: modules/ROOT/pages/tutorials/cashback.adoc [hunk 10]
```diff
@@ -55,25 +55,25 @@ As next step, we want to install the `lisk-sdk` package and add it to our projec
...
+<1> Install the Lisk SDK as a dependency for the node server side.
 <2> Install Lisk Elements dependencies for the client side scripts.
 
 [IMPORTANT]
 ====
-If you haven’t already, you need to create a database before you start.
-The default database name is lisk_dev, so for the development purposes, a command createdb lisk_dev will set you up.
-The default database user and password are lisk and password, they can be both changed in the configuration of Lisk SDK.
+It is necessary to create a database before starting.
+The default database name is lisk_dev, so for the development purposes, a command createdb lisk_dev can be used.
```
Comment (Tschakki) [inline modules/ROOT/pages/tutorials/cashback.adoc]:
```suggestion
The default database name is lisk_dev, so for the development purposes, a command `createdb lisk_dev --owner lisk` can be used.
```

File: modules/ROOT/pages/tutorials/hello-world.adoc [hunk 11]
```diff
@@ -5,23 +5,23 @@ Mona Bärenfänger <mona@lightcurve.io>
 :experimental:
 :v_core: master
 
-Welcome to the step-by-step guide of creating the Hello World application with Lisk Alpha SDK.
-A simple App, showcasing a minimal setup of a blockchain application with 1 xref:customize.adoc[custom transaction] type: the "Hello" transaction.
+Welcome to the step-by-step guide of creating the Hello World application with the Lisk Alpha SDK.
+Firstly, a simple App can be used which showcases a minimal setup of a blockchain application with 1 xref:customize.adoc[custom transaction] type: the "Hello" transaction.
```
Comment (Tschakki) [inline modules/ROOT/pages/tutorials/hello-world.adoc]:
```suggestion
The Hello World application is a simple blockchain app that showcases a minimal setup of a blockchain application with 1 xref:customize.adoc[custom transaction] type: the "Hello" transaction.
```

File: modules/ROOT/pages/tutorials/hello-world.adoc [hunk 12]
```diff
@@ -5,23 +5,23 @@ Mona Bärenfänger <mona@lightcurve.io>
 :experimental:
 :v_core: master
 
-Welcome to the step-by-step guide of creating the Hello World application with Lisk Alpha SDK.
-A simple App, showcasing a minimal setup of a blockchain application with 1 xref:customize.adoc[custom transaction] type: the "Hello" transaction.
+Welcome to the step-by-step guide of creating the Hello World application with the Lisk Alpha SDK.
```
Comment (Tschakki) [inline modules/ROOT/pages/tutorials/hello-world.adoc]:
```suggestion
Welcome to the step-by-step guide of creating the Hello World application with the Lisk SDK.
```

File: modules/ROOT/pages/tutorials/hello-world.adoc [hunk 13]
```diff
@@ -56,25 +56,25 @@ IMPORTANT: If you miss some of the dependencies, please go to xref:setup.adoc#_p
...
+<1> Install the Lisk SDK as dependency for the node side.
+<2> Install Lisk Elements dependencies for the client side scripts.
 
 [IMPORTANT]
 ====
-If you haven’t already, you need to create a database before.
-The default database name is `lisk_dev`, so for the development purposes, a command `createdb lisk_dev` will set you up.
-The default database user and password are `lisk` and `password`, they can be both changed in the configuration of Lisk SDK.
+If a database has not previously been created then it is necessary to create one.
+The default database name is `lisk_dev` The following command,  `createdb lisk_dev` should be used.
```
Comment (Tschakki) [inline modules/ROOT/pages/tutorials/hello-world.adoc]:
```suggestion
The default database name is `lisk_dev` The following command,  `createdb lisk_dev --owner lisk` should be used.
```

**Review by Tschakki — CHANGES_REQUESTED** [review]
Thank you very much!

Only a few small remarks/suggestions from my side :)

File: modules/ROOT/pages/index.adoc (hunk 2 above)
Comment (chris529) [reply modules/ROOT/pages/index.adoc]: Good point Mona.

Comment (chris529) [reply modules/ROOT/pages/index.adoc]: Agreed

**Review by Tschakki — APPROVED** [review]

**Pull request merged**